android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -373,6 +421,17 @@ class XMTPModule : Module() {
             }
         }
 
+        AsyncFunction("verifySignature") Coroutine { inboxId: String, message: String, signature: List<Int> ->
+            withContext(Dispatchers.IO) {
```

Comment:
Suggestion: I see in all other functions we the name of the function. Can we have it here too?
```
logV("verifySignature")
```